### ISSUE TITLE
Layout of plot border with axis.visible = False

### ISSUE THREAD

**Issue by David-Herman** [bokeh/bokeh#7202]
Hello,

I am using Bokeh 12.10 for plotting and have a plot where I have set the axis to be invisible and have tried to remove all extra spacing around the border via the plot.min_border = 0 command. I have search SO for another method of removing the extra white space used for the invisible axis.

The image below illustrates the extra white space retained even after the following configuration below.

```
plot.xaxis.visible = False
plot.yaxis.visible = False
plot.min_border = 0 
```

![image](https://user-images.githubusercontent.com/22305345/32670591-3cd748e6-c612-11e7-8b4e-5620c0333e0c.png)

**Comment by mattpap** [comment]
This was fixed in PR #7008.

**Comment by David-Herman** [comment]
thanks.
